webapp-django/crashstats/signature/views.py

```diff
@@ -135,13 +135,18 @@ def signature_reports(request):
         x for x in data['columns'] if x in allowed_fields
     ]
 
-    params['_columns'] = data['columns']
+    params['_columns'] = list(data['columns'])
 
     # The uuid is always displayed in the UI so we need to make sure it is
     # always returned by the model.
     if 'uuid' not in params['_columns']:
         params['_columns'].append('uuid')
 
+    # The `uuid` field is a special case, it will not be displayed like
+    # normal columns.
+    if 'uuid' in data['columns']:
+        data['columns'].remove('uuid')
+
```

Comment:
This is a mess :)
A couple of lines above it says `if 'uuid' not in params['_columns']`. That changes `_columns` which was created from `columns` and then later it possible changes `columns`. 
This `params` vs. `data` makes my head spin.